ext/xsl/tests/xsltprocessor_hasExsltSupport_not_available_extension.phpt

```diff
@@ -10,4 +10,7 @@ $proc = new XSLTProcessor();
 var_dump($proc->hasExsltSupport());
 ?>
 --EXPECTF--
-Fatal error: Class 'XSLTProcessor' not found in %s on line %d
\ No newline at end of file
+Fatal error: Uncaught Error: Class 'XSLTProcessor' not found in %s:%d
+Stack trace:
+#%d {main}
+  thrown in %s on line %d
```

Comment:
This test makes no sense to me at all. Instead of fixing the expected output, I'd rather rid the test altogether.

Reply:
I thought the same.
@nikic May we remove this one? Already 2 votes :smile:

Reply:
I agree, kill it